lib/plistLib/main.js

```diff
@@ -250,8 +252,8 @@ define([
             dict: function(doc, dict) {
                 var fragment = doc.createDocumentFragment(),
                     key, value, keyElement;
-                for(key in dict) { 
-                    if(!dict.hasOwnProperty(key) || dict[key] === undefined || dict[key] === null)
+                for(key in dict) {
+                    if(!Object.prototype.hasOwnProperty.call(dict, key) || dict[key] === undefined || dict[key] === null)
```

Comment:
If you do `Object.create(null)` you don't have `Object` as prototype and thus not `hasOwnProperty` as a method.